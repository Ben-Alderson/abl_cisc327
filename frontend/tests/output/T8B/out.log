Enter command: Enter login: Enter command: Enter command: Enter the account to withdraw: Enter amount to withdraw: ERROR: You need to login first
Enter command: 