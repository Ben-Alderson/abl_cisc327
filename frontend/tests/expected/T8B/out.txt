error(You must login to withdraw)


